pages/[slug].tsx

```diff
@@ -2,18 +2,25 @@ import { loadProfilePaths, loadSingleProfile } from "../util/load-profiles";
 import ProfileDetail from "../components/profile_detail/personal-card";
 import DetailCard from "../components/profile_detail/detail-card";
 import path from "path";
+import Head from "next/head";
 import { ProfileType } from "../types/ProfileType";
 
 export default function DetailProfile({ profile }: { profile: ProfileType }) {
   return (
-    <div className="no-wrap mt-8 flex flex-col gap-4 md:flex-row">
-      <div className="w-full md:w-1/3">
-        <ProfileDetail profile={profile} />
-      </div>
-      <div className="w-full md:w-3/4">
-        <DetailCard profile={profile} />
+    <div>
+      <Head>
+        <title>Fitness-Accountability</title>
```

Comment:
We could add the username of the persons profile to the title, what do you think?

Reply:
Yes!